src/app_engine/apprtc.py

```diff
@@ -534,8 +534,10 @@ def write_response(self, target_page, params={}):
 
   def get(self):
     """Renders index.html."""
-    if self.request.headers['Host'] == 'apprtc.net':
-      webapp2.redirect('https://www.apprtc.net', permanent=True)
+    REDIRECT_DOMAINS = ['apprtc.appspot.com', 'apprtc.net', 'www.apprtc.net',
+        'apprtc.webrtc.org', 'www.appr.tc']
+    if self.request.headers['Host'] in REDIRECT_DOMAINS:
```

Comment:
True, but hopefully use of apprtc.net should be rare.

I do think we want to ensure that at least apprtc.appspot.com/r/foo redirects properly.